tests/test_data_frame.py

```diff
@@ -584,5 +584,45 @@ def test_memory_usage(self):
                          base_df.memory_usage(deep=True, index=True).sum())
 
 
+    def test_attr_access(self):
+        base_df = pd.DataFrame([{'n': i, 's': (i*2)} for i in range(10)])
+        df = ps.DataFrame(base_df)
+
+        assertDataFrameEqualsPandas(df.n, pd.DataFrame(base_df.n))
+
+    def test_complex_write_attr_access(self):
```

Comment:
I think this is way to complex of a test. Probably best to do a simpler write test for attr, just a few lines.